packages/apps/dev-wallet/src/pages/sig-builder/style.css.ts

```diff
@@ -0,0 +1,6 @@
+import { style } from '@vanilla-extract/css';
+
+export const codeArea = style({
+  width: `100%`,
```

Comment:
do we need this one? could be solved with flexing the parent, i can take a look in another pr

Reply:
I just focused on functionality in this PR
the whole page need to be refactored. its better to do it when we have design for it.